admin/server/alerts.go

```diff
@@ -444,6 +444,8 @@ func (s *Server) yamlForManagedAlert(opts *adminv1.AlertOptions, alertName, owne
 	res := alertYAML{}
 	res.Kind = "alert"
 	res.Title = opts.Title
+	// manually add the ref
+	res.Refs = []string{fmt.Sprintf("dashboard/%s", opts.DashboardName)}
```

Comment:
Can we put `MetricsView/%s` here instead (already supported by the parser)? There's some talk about changing "dashboard" to mean custom dashboards, so would be nice not to have to migrate the virtual alert artifacts